replay.pokemonshowdown.com/src/replays-battle.tsx

```diff
@@ -494,7 +494,7 @@ export class BattlePanel extends preact.Component<{id: string}> {
         {/* {} <code>{this.keyCode}</code> */}
       </p> : <p>&nbsp;</p>}
       {!PSRouter.showingLeft() && <p>
-        <a href="." class="button"><i class="fa fa-caret-left"></i> More replays</a>
+        <a href={`./${PSRouter.leftLoc || ''}`} class="button"><i class="fa fa-caret-left"></i> More replays</a>
```

Comment:
Good catch! But probably more complex than it needs to be.

```suggestion
        <a href={PSRouter.leftLoc || '.'} class="button"><i class="fa fa-caret-left"></i> More replays</a>
```

Reply:
Are you sure this change is correct? This is what I tried at first but it kept appending the query to the end of the replay link rather than going back to the base link of replay.pokemonshowdown.com.